### ISSUE TITLE
Add 'type' in before import statements

### ISSUE THREAD

**Issue by johan-timberit** [jburzynski/TypeGen#161]
Hi!
First of all, really go TS-generator! 

I have a question though: 
Would it be possible to add 'type' to auto imported classes/interfaces genreted by typeGen?

Message in VS Code:
"This import is never used as a value and must use 'import type' because 'importsNotUsedAsValues' is set to 'error'.ts(1371)"

Incorrect (generated by typeGen):
```typescript
/**
 * This is a TypeGen auto-generated file.
 * Any changes made to this file can be lost when this file is regenerated.
 */

import { OrderNumber } from "./order-number";

export interface OrderNumberCreateCommand {
    orderNumberEntity: OrderNumber;
}
```

Correct (edited by myself)
```typescript
/**
 * This is a TypeGen auto-generated file.
 * Any changes made to this file can be lost when this file is regenerated.
 */

import type { OrderNumber } from "./order-number";

export interface OrderNumberCreateCommand {
    orderNumberEntity: OrderNumber;
}

```

**Comment by jburzynski** [comment]
Hi, I'm glad it's useful! I added this functionality to version 4.4.0. It can be enabled with the [`useImportType`](useImportType) option (or `UseImportType` in `GeneratorOptions`). It's disabled by default.

**Comment by johan-timberit** [comment]
Hi!
Great! What would be the best way of using GeneratorOptions? I am simply generating on build (not programatically with API) and adding attributes on the classes I want generate. 

Can GenerateOptions for example be added to a config file? For example "dotnet-typegen generate -c "path_to_config_file"? 

![image](https://github.com/jburzynski/TypeGen/assets/68020044/6e68b1f1-3e97-424d-b323-6b96394a57e8)

Br
Johan

**Comment by johan-timberit** [comment]
Have I understodd i correctly that if I put a file in the root of the project, naming it "tgconfig.json", it will automatically used by typeGen?

I created such file and added 

`{
  "useImportType": true
}
`

Nothing happens when this is added. 

Br
Johan

**Comment by jburzynski** [comment]
There was a bug, it should be fixed in 4.4.1 and the config above should work.
